baybe/recommenders/meta/sequential.py

```diff
@@ -26,25 +29,25 @@
 
 
 @define
-class TwoPhaseStrategy(Strategy):
-    """A two-phased strategy that switches the recommender at a certain specified point.
+class TwoPhaseMetaRecommender(MetaRecommender):
+    """A two-phased meta recommender that switches at a certain specified point.
```

Comment:
"Switches the used pure recommender" or something like that, otherwise it is not clear what "switches" refers to.

Reply:
I attempted to add this `A two-phased meta recommender that switches between two pure recommenders at a
    certain specified point.` but that lines too long. At the same time one cannot break the summary line hence my hands are tied

tbh in the context that meta recommenders select pure recommenders I think `switch` is quite clear